script/release/ci-release-build.js

```diff
@@ -14,15 +23,23 @@ const appVeyorJobs = {
 };
 
 const circleCIPublishWorkflows = [
-  'linux-publish',
-  'macos-publish'
+  'macos-publish',
```

Comment:
This change seems unnecessary

Reply:
🤦‍♀️ Yep, this was a cherry-pick mistake on my part, I removed `macos-publish` in the original GH Actions PR and added it back in here in the wrong order. I'll update so there's no change here, thanks!